src/compiler/utilitiesPublic.ts

```diff
@@ -2334,7 +2321,6 @@ function isStatementKindButNotDeclarationKind(kind: SyntaxKind) {
         || kind === SyntaxKind.MergeDeclarationMarker;
 }
 
-/** @internal */

```

Comment:
On second thought, `isDeclaration` contains `.parent` references, so we should probably leave it internal-only for now.

Reply:
I'll revert it, thanks.

Reply:
Done.